scripts/quickstart.sh

```diff
@@ -606,7 +606,7 @@ function run {
         --restart=always \
         $DOCKER_IMAGE_PARITY \
         --role node \
-        --parity-args --no-warp --node-key $NODE_KEY --jsonrpc-threads $NUM_RPC_THREADS --jsonrpc-server-threads $NUM_HTTP_THREADS
```

Comment:
Why we are changing this?

Reply:
No longer supported in OE

Reply:
But we are still want to support parity in the quickstart

Reply:
Also need to delete from this place also https://github.com/fuseio/fuse-network/blob/fe2ed591d360fb14dada64c268f85ed686a2fee2/scripts/quickstart.sh#L580 ?

Reply:
Yes this is redundant now

Reply:
fixed at f106776